css/selectors/xr-overlay.json

```diff
@@ -0,0 +1,46 @@
+{
+  "css": {
+    "selectors": {
+      "xr-overlay": {
+        "__compat": {
+          "description": "`:xr-overlay`",
+          "spec_url": "https://immersive-web.github.io/dom-overlays/#selectordef-xr-overlay",
+          "tags": [
+            "web-features:webxr-dom-overlays"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "83"
```

Comment:
✅ https://github.com/chromium/chromium/commit/1085a76adef5254a53ee941f4c0b5d0420049521 implemented the feature in [82.0.4065.0](https://github.com/chromium/chromium/releases/tag/82.0.4065.0) before 83.0.4103.

Bug: https://issues.chromium.org/issues/41474948